openfl-gramine/Dockerfile.gramine

```diff
@@ -0,0 +1,38 @@
+ARG BASE_IMAGE=python:3.8
+FROM ${BASE_IMAGE}
+
+# install openfl
+# RUN --mount=type=cache,target=/root/.cache/ \
+#     pip install --upgrade pip && \
+#     pip install openfl
+
+# for now install from source (branch with logs_writer turned off)
+RUN git clone https://github.com/igor-davidyuk/openfl.git --branch gramine-rebased
+WORKDIR /openfl
+RUN --mount=type=cache,target=/root/.cache/ \
+    pip install --upgrade pip && \
+    pip install .
+WORKDIR /
+
+# install gramine
+RUN curl -fsSLo /usr/share/keyrings/gramine-keyring.gpg https://packages.gramineproject.io/gramine-keyring.gpg && \
+    echo 'deb [arch=amd64 signed-by=/usr/share/keyrings/gramine-keyring.gpg] https://packages.gramineproject.io/ stable main' | \
+    tee /etc/apt/sources.list.d/gramine.list
+RUN --mount=type=cache,id=apt-dev,target=/var/cache/apt \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+    gramine libprotobuf-c-dev
+    # there is an issue for libprotobuf-c in gramine repo, install from apt for now
+
+# graminelibos is under this dir
+ENV PYTHONPATH=/usr/local/lib/python3.8/site-packages/:/usr/lib/python3/dist-packages/:
```

Comment:
This path might be different for different users. Maybe we could add a note for users to know that we are using this as PYTHONPATH.

Reply:
This path is actually fixed as it Is inside the container (or image 🤷‍♂️)